cli/src/command/host.rs

```diff
@@ -137,9 +137,8 @@ fn setup_crate(host_path: PathBuf, rev: Option<String>, tag: Option<String>) ->
         fp2,
         concat!(
             "\n",
-            "# Generated by cargo-nexus, do not remove!\n",
```

Comment:
Oh I would leave this line in, just remove the one beneath about "This profile".

Reply:
done in 99bd28e